pkgs/by-name/gt/gtklock/package.nix

```diff
@@ -0,0 +1,75 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  scdoc,
+  pkg-config,
+  wrapGAppsHook3,
+  gtk3,
+  pam,
+  gtk-session-lock,
+  runCommand,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "gtklock";
+  # Must run nixpkgs-review between version changes
```

Comment:
What you actually mean is that we want to check the plugins with nixpkgs-review, right?

Reply:
Oh I forgot `nixpkgs-review` actually doesn't run `passthru.tests`